modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java

```diff
@@ -844,6 +844,17 @@ public void testAddOrUpdateCustomObjectField() throws Exception {
 				objectField1.getExternalReferenceCode()
 			).build());
 
+		_testAddOrUpdateCustomObjectField(
```

Comment:
It seems that this is not the right place to test the bug situation. 
Here you are testing to add an object field with different ERCs and the same name.
In the bug situation they are passing object fields with different names but the same ERC.